simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/CardCreatorActivity.kt

```diff
@@ -16,5 +17,14 @@ class CardCreatorActivity : AppCompatActivity() {
     super.onCreate(savedInstanceState)
     binding = ActivityCardCreatorBinding.inflate(layoutInflater)
     setContentView(binding.root)
+    if (intent.extras.getBoolean("isLoggedIn")) {
+      Cache(this).isJuvenileCard = true
+      binding.toolbarTitleTv.text = getString(R.string.create_child_card)
```

Comment:
You should be able to omit the getString call here. The TextView class includes a setText method that accepts a resource id value:

https://developer.android.com/reference/kotlin/android/widget/TextView#settext_3